tests/labels/test_api.py

```diff
@@ -251,6 +251,11 @@ async def test_is_valid_hex_color(value, spawn_client, resp_is):
     if value == "valid_hex_color":
         await resp_is.not_found(resp)
     else:
-        await resp_is.invalid_input(
-            resp, {"color": ["This is not a valid Hexadecimal color"]}
-        )
+        assert resp.status == 400
+        assert await resp.json() == [{
+            "loc": ["color"],
+            "msg": 'string does not match regex "^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$"',
+            "type": "value_error.str.regex",
+            "ctx": {"pattern": '^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$'},
+            "in": "body"
+        }]
```

Comment:
This is uninformative and hard to read. The old error message was better.

I liked what you did in core for this using the validators. I have generalized that in [`virtool_core.models`](https://github.com/virtool/virtool-core/blob/211949cf2c2088ca9ee0e6399b1d24481a91c2d2/virtool_core/models/__init__.py#L4).

Can you use this and see if it produces a better error message.